bundle/language_en.properties

```
@@ -663,38 +664,72 @@ angal.hospital.thevisitinghourvaluesmustbeintherange0to24.msg
-angal.inventoryrow.doyoureallywanttodeletethisinventoryrow.msg                                         = Do you really want to delete this inventory row?
+angal.inventory.savesuccess.msg                                                                        = Inventory saved.
+angal.inventory.status.label                                                                           = Status: 
+angal.inventory.new.col                                                                                = New
+angal.inventory.lot.btn                                                                                = Lot
+angal.inventory.lot.btn.key                                                                            = L
+angal.inventory.supplier.label                                                                         = Supplier
+angal.inventory.error.msg                                                                              = Error during some products's recording.
+angal.inventoryrow.addnotyetlistedproducts.title                                                       = Add products
+angal.inventoryrow.doyoureallywanttodeletethisinventoryrow.msg                                         = Do you really want to delete this/those inventory(ies) row(s)?
+angal.inventoryrow.doyoureallywanttocleanthistable.msg                                                 = Do you really want to clean this table?
+angal.inventoryrow.doyouwanttoaddallnotyetlistedproducts.msg                                           = Do you want to add all not yet listed products?
+angal.inventoryrow.doyouwanttocontinueediting.msg                                                      = Do you want to continue editing?
+angal.inventoryrow.doyouwanttosavethechanges.msg                                                       = Do you want to save the changes?
+angal.inventoryrow.notfound.msg                                                                        = Inventory not found.
+angal.inventoryrow.invalidquantity.msg                                                                 = Invalid quantity.
 angal.inventoryrow.pleaseselectonlyoneinventoryrow.msg                                                 = Please select only one inventory row.
+angal.inventoryrow.pleaseselectatleastoneinventoryrow.msg                                              = Please select at least one inventory row.
+angal.inventoryrow.pleaseselectoneinventoryrow.msg                                                     = Please select one inventory row.
+angal.inventoryrow.thislotcodealreadyexists.msg                                                        = This lot code already exists. Please change it.
+angal.inventory.inventoryisalreadysaved.msg                                                             = This inventory is already saved. Please enter at least one inventory row before continue.
```

Update occurences in the code also

```suggestion
angal.inventory.inventoryisalreadysaved.msg                                                            = This inventory is already saved. Please enter at least one inventory row before continue.
```

---

I already update occurences in the code